lib/data_update_scripts/20210622145212_backfill_forem_consumer_app_team_id.rb

```diff
@@ -0,0 +1,9 @@
+module DataUpdateScripts
+  class BackfillForemConsumerAppTeamId
+    def run
+      forem_app = ConsumerApp.find_by(app_bundle: ConsumerApp::FOREM_BUNDLE, platform: :ios)
```

Comment:
If we do this in SQL we don't have to worry about `nil` and the `&.` on the next line. I'd probably also re-use `ConsumerApp::FOREM_TEAM_ID` here:

```ruby
ConsumerApp
  .where(app_bundle: ConsumerApp::FOREM_BUNDLE, platform: :ios)
  .update(team_id:  ConsumerApp::FOREM_TEAM_ID)
```

Since there are no validations on the `team_id` attribute and we're not setting anything else I don't think we need to instantiate an object here.

Reply:
+1